docs/content/deployment/run-retries.mdx

```diff
@@ -5,19 +5,24 @@ description: Automatically retry Dagster runs
 
 # Run Retries
 
-If you configure retries at the [Job](/\_apidocs/jobs#jobs) level, a new run will be kicked off when a run for that job fails. Compared to [Op retries](/concepts/ops-jobs-graphs/op-retries), the max retry limit for run retries applies to the whole run instead of each individual Op. Run retries also handle the cases where a run worker crashed.
+If you configure run retries, a new run will be kicked off whenever a run fails for any reason. Compared to [Op retries](/concepts/ops-jobs-graphs/op-retries), the maximum retry limit for run retries applies to the whole run instead of each individual op. Run retries also handle the case where the run process crashes or is unexpectedly terminated.
 
 ## Configuration
 
-To enable run retries, add the following to your `dagster.yaml`. This will start a new daemon which polls to the event log for run failure events.
+How to configure run retries depends on whether you're using Dagster Cloud or Dagster Open Source:
 
-```yaml file=/deploying/dagster_instance/dagster.yaml startafter=start_run_retries endbefore=end_run_retries
+- **Dagster Cloud**: Use the [Dagster Cloud UI or the dagster-cloud CLI](/dagster-cloud/managing-deployments/managing-deployments#configuring-deployment-settings) to set a default maximum number of retries. Run retries do not need to be explicitly enabled.
```

Comment:
i was able to make mdx-format happy by using the URL instead of the 
`
[cloud-deployment-settings]: /dagster-cloud/managing-deployments/managing-deployments#configuring-deployment-settings
`
thing that the concurrency page uses

Reply:
So, so weird. I can't wait til' we're rid of this. Glad you got it working tho!